### ISSUE TITLE
Each component should accept all html props

### ISSUE THREAD

**Issue by JasonStoltz** [elastic/search-ui#241]
https://reakit.io/docs/styling/

Accepting all HTML props makes it much easier to customize the components in the library.

Example:

In addition to the required and known parameters that PagingInfo accepts, it should also pass through any other prop that it receives:

```
<PagingInfo style={{display: "flex"}} />
```

To make this change, in the `PagingInfo.js` container, you need to make sure to pass through the additional props to the corresponding view:

`react-search-ui - PagingInfo.js`
```
const {
      className,
      current,
      resultsPerPage,
      resultSearchTerm,
      totalResults,
      view,
      ...rest
    } = this.props;

return View({
      className,
      end: end,
      searchTerm: resultSearchTerm,
      start: start,
      totalResults: totalResults,
      ...rest
    });
```

In the corresponding view, you would render those additional props to the root element:

`react-search-ui-views - PagingInfo.js`
```
function PagingInfo({ className, end, searchTerm, start, totalResults, ...rest }) {
  return (
    <div className={appendClassName("sui-paging-info", className)} {...rest}>

    </div>
  );
}
```

**Comment by Jfelix61** [comment]
I can do this.

**Comment by JasonStoltz** [comment]
@Jfelix61 Great! Feel free to work on this and submit a PR, and let us know if you have any questions.

**Comment by Jfelix61** [comment]
Submitted the PR. However circle-ci tests fail since the SearchBox component receives new props: result of the rest implementation.

**Comment by JasonStoltz** [comment]
Closed by #368
